dirsrvtests/tests/suites/plugins/modrdn_test.py

```diff
@@ -0,0 +1,173 @@
+# --- BEGIN COPYRIGHT BLOCK ---
+# Copyright (C) 2025 Red Hat, Inc.
+# All rights reserved.
+#
+# License: GPL (version 3 or any later version).
+# See LICENSE for details.
+# --- END COPYRIGHT BLOCK ---
+#
+import pytest
+from lib389.topologies import topology_st
+from lib389._constants import DEFAULT_SUFFIX
+from lib389.idm.group import Groups
+from lib389.idm.user import nsUserAccounts
+from lib389.plugins import (
+    AutoMembershipDefinitions,
+    AutoMembershipPlugin,
+    AutoMembershipRegexRules,
+    MemberOfPlugin,
+    ReferentialIntegrityPlugin,
+)
+
+pytestmark = pytest.mark.tier1
+
+USER_PROPERTIES = {
+    "uid": "userwith",
+    "cn": "userwith",
+    "uidNumber": "1000",
+    "gidNumber": "2000",
+    "homeDirectory": "/home/testuser",
+    "displayName": "test user",
+}
+
+
+def test_modrdn_of_a_member_of_2_automember_groups(topology_st):
+    """Test that a member of 2 automember groups can be renamed
+    :id: 0e40bdc4-a2d2-4bb8-8368-e02c8920bad2
```

Comment:
```suggestion
    """Test that a member of 2 automember groups can be renamed
    
    :id: 0e40bdc4-a2d2-4bb8-8368-e02c8920bad2
```
Small change to make test metadata validator happy.